### PR TITLE
Update simulated motion sensors in ST when they update in HA

### PR DESCRIPTION
**Pull request by mattsch**

### CODE REVIEW

**Comment by stjohnjohnson** [comment]
Hmm, it's not in the documented capabilities: http://docs.smartthings.com/en/latest/capabilities-reference.html#motion-sensor

I don't have a motion sensor, but does this work for you?

**Comment by mattsch** [comment]
Yup, I've created two simulated motion sensors in ST and automations in HA to publish 'active' or 'inactive'.  The changes are reflected on the ST side and the IDE logs show it calling the proper active() or inactive() methods for the simulated devices.

File: smartapps/stj/mqtt-bridge.src/mqtt-bridge.groovy [hunk 1]
```diff
@@ -603,6 +604,14 @@ def actionOnOff(device, attribute, value) {
     }
 }
 
+def actionActiveInactive(device, attribute, value) {
+    if (value == "active") {
+            device.active()
```
Comment (stjohnjohnson) [inline smartapps/stj/mqtt-bridge.src/mqtt-bridge.groovy]: Indentation seems a little high here

**Comment by stjohnjohnson** [comment]
I wonder if this will cause problems if the functions don't exist

File: smartapps/stj/mqtt-bridge.src/mqtt-bridge.groovy (hunk 1 above)
Comment (mattsch) [inline smartapps/stj/mqtt-bridge.src/mqtt-bridge.groovy]: Good catch, fixed.

**Comment by mattsch** [comment]
I would expect we'd just see an error in the logs, but this is ST so I can't be sure.  Those functions should always exist for motion detectors from what I've been able to glean from the interwebs.

**Comment by stjohnjohnson** [comment]
LGTM then 👍 @jer?

**Comment by jer** [comment]
👍

**Pull request merged**